projects/ngrx-clean-forms/src/lib/directives/controls/abstract-control.directive.ts

```diff
@@ -52,6 +52,13 @@ export abstract class AbstractControlDirective<T> implements OnDestroy {
     }
 
     @Output() controlUpdate = new EventEmitter<FormControlUpdate<T>>(true);
+    controlSummary$ = new Subject<FormControlSummary<T>>();
+
+    get controlSummary() {
+        return this._controlSummary;
+    }
+
+    private _controlSummary: FormControlSummary<T>;
```

Comment:
Probably only the observable should be public and the subject should be private. So maybe like this instead:

```ts
private controlSummarySubject = new Subject<FormControlSummary<T>;
readonly controlSummary$ = this.controlSummarySubject.asObservable();
```

Reply:
@NiklasPor done. I kept the `_` prefix because it is already used for the controlSummary. I'd rather change both names in an additional commit, if you don't like the dash.